core/models/user.py

```diff
@@ -95,9 +95,11 @@ class UserFeedback(models.Model):
     """ Stores user feedback data """
     FEEDBACK = "feedback"
     BUG = "bug"
+    FEATURE_REQUEST = "feature_request"
     option = (
         (FEEDBACK, "feedback"),
         (BUG, "bug"),
+        (FEATURE_REQUEST, "feature_request"),
     )
     reported_by = models.ForeignKey(User, on_delete=models.PROTECT)
     message = models.CharField(max_length=200, null=False)
```

Comment:
Also, update the `message` field to a [TextField](https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.TextField) to accommodate reports that may exceed the 200 character limit currently set on the field.